src/main/java/HW13/MySQLConnection.java

```diff
@@ -0,0 +1,20 @@
+package HW13;
+
+import org.apache.log4j.Logger;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class MySQLConnection {
```

Comment:
As it is a util class better to make it final and create a private constructor to prevent the creation of an object of this class.